### ISSUE TITLE
[Bug]: Error on Startup in FreeTube Version v0.19.0-beta - Issue Reopened

### ISSUE THREAD

**Issue by mathe00** [FreeTubeApp/FreeTube#3936]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

I am encountering an issue at the startup of FreeTube version v0.19.0-beta. When attempting to launch the application, an error window pops up, preventing me from accessing the application. This is the only problem I am experiencing. The other details in the previous report do not apply to my situation.

**Steps to Reproduce**

1. Attempt to launch FreeTube v0.19.0-beta.
2. An error window immediately appears upon startup, preventing access to the application.

**Notes**

I want to clarify that this issue occurs during the application's startup. Unlike the previous report, I cannot access the application interface. This situation renders the normal use of FreeTube impossible.

I kindly request the development team to reopen this issue for investigation and resolution of this startup problem, allowing users to enjoy the application without encountering blockages.

Thank you for taking these details into consideration.

### Expected Behavior

I expected FreeTube v0.19.0-beta to launch without any errors, allowing me to access and use the application normally.

### Issue Labels

causes crash, inconsistent behavior, usability issue

### FreeTube Version

v0.19.0-beta

### Operating System Version

Manjaro Linux, Kernel 6.1.44-1-MANJARO (64-bit)

### Installation Method

AUR

### Primary API used

Local API

### Additional Information

The issue occurs at the application's startup, displaying an error window that prevents me from accessing FreeTube. This situation renders the application unusable. Here is the link to the related original issue: [GitHub Issue Link](https://github.com/FreeTubeApp/FreeTube/issues/1250)

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
You ticked the box saying you encountered it in one of the official releases, but then listed your installation method as AUR, which is an unofficial release. Did you actually test it on an official release too or just AUR?

Additionally please provide a screenshot of the error, we can't really investigate anything without any information to go off.

**Comment by mathe00** [comment]
![image](https://github.com/FreeTubeApp/FreeTube/assets/97852395/f942eef9-3a51-4541-9bd9-a10de15012fc)
Since my installation with AUR

![image](https://github.com/FreeTubeApp/FreeTube/assets/97852395/4ae5cbd1-70e4-49e4-8f8c-ed365bb7cba7)
By launching the AppImage just after

I actually installed it from the AUR, apologies for the error. I've just downloaded the AppImage and encountered the same error as when trying to launch Freetube installed with the AUR.

**Comment by absidue** [comment]
As you seem to be the only person with this issue at the moment, this doesn't sound like an issue in FreeTube itself, as the users on the previous issue seemed to be using arch/manjaro too, I suspect this is an operating/setup specific issue (never had anything like that in my years of using FreeTube on Windows).

Could you try checking if you have any instances of FreeTube running in your task manger/process viewer, if yes kill them and try again?

**Comment by mathe00** [comment]
![image](https://github.com/FreeTubeApp/FreeTube/assets/97852395/556f0386-8f76-4352-bad6-3f5489007acb)

Indeed, when I launch FreeTube, I have all these processes running. And when I close FreeTube, only one of these processes closes. And to answer your first question, no I hadn't uninstalled the AUR version before trying AppImage, please excuse me.

**Comment by Garpgarp** [comment]
Hi, non-technical user here that had presumably a very similar issue on Fedora, but I hope it can help the OP.

Managed to get things up and running by making backups of whatever Freetube labelled folders I could find under home directory (~/.var/io.freetubeapp.FreeTube and ~/.config/Freetube) then deleting those folders, and running the following commands based on what I could find on Google:

dnf history userinstalled
dnf remove freetube-0.19.0-1.x86_64 

Opted to reinstall using the official flatpak and after reimporting my history from the folder backups we're up and running.

![Screenshot from 2023-08-24 23-20-32](https://github.com/FreeTubeApp/FreeTube/assets/70253673/ecb80633-efb4-4767-8766-ebb2e789e2f2)
![Screenshot from 2023-08-24 23-20-24](https://github.com/FreeTubeApp/FreeTube/assets/70253673/c1d9a47a-8b21-44a0-9951-22b53f1ae300)

**Comment by absidue** [comment]
Very odd that it is happening but glad that a full reinstall fixes it.

Sounds like thing get handled properly with the flatpak version but fail when running without flatpak. Makes me think the issue is outside of FreeTube but this is Linux, so it could be caused by anything.

**Comment by mathe00** [comment]
I had the problem again with another fresh installation of Linux Manjaro, but this time no installation via the AUR, it's the AppImage installation. I'm just saying that to let everyone know that it's not AUR specific. So I do `pkill freetube` as soon as I encounter this problem, it's a quick and efficient way to temporarily fix the problem.

**Comment by dcstraney** [comment]
Encountered this same error in Windows 11 / x64 with 0.19.1.  Sometimes would happen randomly and after re-trying later would work correctly, but today was consistently showing the same "object destroyed" message each time.  Installing the latest nightly build (3610) fixed it, but will see how long that lasts, or if that was just a roundabout fix to a "leftover processes" issue (tried this before seeing this issue).  If I see the same error again will have to check the process list - will report back.

**Comment by dcstraney** [comment]
With nightly build 3610, however, I did still see FreeTube processes after closing the FreeTube window:
![image](https://github.com/FreeTubeApp/FreeTube/assets/63062249/bcb53e6e-5e7a-4f99-a91e-7b9d56f75d56)
FreeTube still launched a second and third time correctly though, and both times after closing the window, all its processes disappeared over the course of ~10 seconds, whether on the Subscriptions screen or in the middle of a video - so this seems to be intermittent.

**Comment by PABERTHIER** [comment]
Having the same issue with a fresh install from yesterday (latest version).
Windows 10, Version 22H2 (OS Build 19045.4046).

Once the soft has been shut down, no way to reopen it without a computer restart.
I understand the thing about processes, but maybe something has be to be done about handling them to prevent these issues and without risking the data corruption.
Or maybe at least something to warn the user for example ?

PS: Insane work for this software, love it so far ! U guys are monsters !

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@mathe00 do you still face this issue on V0.20.0?

**Comment by mathe00** [comment]
> @mathe00 do you still face this issue on V0.20.0?

No, I haven't had any problems whatsoever with Freetube for a long time now.

**Comment by demidovia** [comment]
Greetings,
Like OP I did search for solution, but I have the same issue with versions 0.20 or 0.21 (the latest I just grabbed from official website). Screenshot is attached.
![Screenshot from 2024-06-24 22-48-44](https://github.com/FreeTubeApp/FreeTube/assets/34301092/c19e3e28-a007-4663-ad64-03d1dbc522ba)
Thank you.

**Comment by PABERTHIER** [comment]
Hi @demidovia !
I had the same issue as you, on the first day I've installed the app.
After a computer restart, never happened again since 20th of feb.
Maybe, each first time of a new install, this bug is happening and then, after a computer restart, it vanishes.

**Comment by demidovia** [comment]
> Hi @demidovia ! I had the same issue as you, on the first day I've installed the app. After a computer restart, never happened again since 20th of feb. Maybe, each first time of a new install, this bug is happening and then, after a computer restart, it vanishes.

I've tried to restart, but the result is the same. Weird thing is. I do not get any errors besides blank black screen in the Freetube window if there is no Internet connection available. Maybe it is tied to my "account" files stored at my Linux PC. I will research how to delete those and try again and report back.

**Comment by demidovia** [comment]
Actually it goes like this: I restart my PC-->Start the app-->Black blank screen in the app window-->I close the app and restart it--->Java error.
Not sure what to do with it.

**Comment by Vinayak-kadu** [comment]
> Actually it goes like this: I restart my PC-->Start the app-->Black blank screen in the app window-->I close the app and restart it--->Java error. Not sure what to do with it.

try this approach . [https://github.com/FreeTubeApp/FreeTube/discussions/5369](Blank Screen)

**Comment by demidovia** [comment]
> > Actually it goes like this: I restart my PC-->Start the app-->Black blank screen in the app window-->I close the app and restart it--->Java error. Not sure what to do with it.
> 
> try this approach . [https://github.com/[/discussions/5369](https://github.com/FreeTubeApp/FreeTube/discussions/5369)](Blank Screen)

Thank you. But can you help me locate the file they are talking about?
